roles/splunk/tasks/configure_bash.yml

```diff
@@ -10,3 +10,4 @@
   loop:
     - { bashtemplate: 'bashrc.j2', bashfilepath: '.bashrc' }
     - { bashtemplate: 'bash_profile.j2', bashfilepath: '.bash_profile' }
+  changed_when: False
```

Comment:
This doesn't really fix idempotency for this task. It just tells Ansible to report that nothing changed even if it did make changes to these files.

Reply:
I tested solely the `append: true` part of this PR without the change to `configure_bash.yml` and was able to confirm that it resolves the idempotency issue for both tasks. Please remove this change from `configure_bash.yml` and we'll get your PR merged. Thank you!

Reply:
oups, it tested but obviously i'm unable to read "OK" vs "CHANGED" 👎 
I'll update the PR

Reply:
And in fact without my changes, i don't reproduce the issue for the `configure_bash.yml`
First execution : 
![image](https://user-images.githubusercontent.com/5929339/110086041-2be42000-7d92-11eb-9532-2e42094a22eb.png)

Second execution : 
![image](https://user-images.githubusercontent.com/5929339/110086094-3acad280-7d92-11eb-9b26-e0e3646e643b.png)